code/modules/shuttle/computer.dm

```diff
@@ -62,8 +62,8 @@
 				to_chat(usr, "<span class='warning'>Shuttle already in transit.</span>")
 				return
 		if(!(href_list["move"] in params2list(possible_destinations)))
-			log_admin("[usr] attempted to href dock exploit on [src] with target location \"[href_list["move"]]\"")
-			message_admins("[usr] just attempted to href dock exploit on [src] with target location \"[href_list["move"]]\"")
+			log_admin("[usr] attempted to forge a target location through a href exploit on [src] with target location \"[href_list["move"]]\"")

```

Comment:
I believe the thing with this was to remove `[href_list["move"]]` since this is the bit the exploiter can edit and type whatever they want with.

Reply:
(Also this code is also in the custom shuttle console too, so that would need copy and pasting away too)

Reply:
Maybe. Looks like it when I actually think about it.

Reply:
looking back, the top one logged to game logs, but you could still input some dodgey things there, so it's probably best its not included in either

Reply:
The problem is that the message to the admins was not sanitized. The logs are sanitized, and it's useful to know what they are sending in general.
Then again, you can check the href logs as well.

Alternatively you can sanitize the value sent to the admins.

Reply:
I see, fair enough. As long as either way people can't send messages that are millions of characters long and fill up the drives it's cool